src/Umbraco.Cms.Integrations.Automation.Zapier/Controllers/PollingController.cs

```diff
@@ -12,20 +13,21 @@
 using Umbraco.Cms.Core.Web;
 using Umbraco.Cms.Web.Common.Controllers;
 using Umbraco.Cms.Core.Services;
-using Umbraco.Cms.Core.Models.PublishedContent;
-using Umbraco.Extensions;
 #else
 using System.Configuration;
 
 using Umbraco.Web;
 using Umbraco.Web.WebApi;
 using Umbraco.Core.Services;
-using Umbraco.Core.Models.PublishedContent;
-
 #endif
 
 namespace Umbraco.Cms.Integrations.Automation.Zapier.Controllers
 {
+    /// <summary>
+    /// When a Zapier user creates a "New Content Published" triggered, he is authenticated, then selects a content type, the API provides an output json with the
```

Comment:
"he is" --> "they are", "then selects" --> "then select"

Reply:
Updated.